src/components/item/sharing/csvImport/DisplayInvitationSummary.tsx

```diff
@@ -0,0 +1,109 @@
+import { Alert, AlertTitle, Stack, Typography } from '@mui/material';
+
+import { Invitation, ItemMembership, PermissionLevel } from '@graasp/sdk';
+import { FAILURE_MESSAGES } from '@graasp/translations';
+
+import { AxiosError } from 'axios';
+
+import { useBuilderTranslation, useMessagesTranslation } from '@/config/i18n';
+import { SHARE_CSV_TEMPLATE_SUMMARY_CONTAINER_ID } from '@/config/selectors';
+import { BUILDER } from '@/langs/constants';
+import { getErrorFromPayload } from '@/utils/errorMessages';
+
+const ErrorDisplay = ({ errorMessage }: { errorMessage: string }): string => {
+  const { t } = useBuilderTranslation();
+  const { t: translateMessages } = useMessagesTranslation();
+  switch (errorMessage) {
+    case 'MODIFY_EXISTING': {
+      return t(
+        'You are trying to import a user that already has a permission on this item. We currently do not support this. Please remove users that already have permissions on this item from your csv file and try again.',
+      );
+    }
+    default: {
+      return translateMessages(FAILURE_MESSAGES.UNEXPECTED_ERROR);
+    }
+  }
+};
+
+const LineDisplay = ({
+  email,
+  permission,
+}: {
+  email: string;
+  permission: PermissionLevel;
+}) => (
+  <Stack direction="row" gap={1}>
+    <Typography>{email}</Typography>
+    <Typography>{permission}</Typography>
+  </Stack>
+);
+
+type Props = {
+  userCsvData?:
+    | { memberships: ItemMembership[]; invitations: Invitation[] }
+    | {
+        groupName: string;
+        memberships: ItemMembership[];
+        invitations: Invitation[];
+      }[];
+  error: Error | null | AxiosError;
+};
+const DisplayInvitationSummary = ({
```

Comment:
I'm not sure to understand how it can have no errors but no userCsvData too ?

Reply:
So the component is rendered without checking if the `userCsvData` is actually present, hence it can be undefined. And the error is always either null (no error) or an error.

Reply:
Mmh ok I see, thanks !